src/libertem/web/cli.py

```diff
@@ -4,17 +4,18 @@
 @click.command()
 @click.option('-p', '--port', help='port on which the server should listen on',
               default=9000, type=int)
-@click.option('-l', '--local-directory', help='local directory to manage dask-worker-space files',
+@click.option('-d', '--local-directory', help='local directory to manage dask-worker-space files',
               default='dask-worker-space', type=str)
 @click.option('-b/-n', '--browser/--no-browser',
               help='enable/disable opening the browser', default='True')
+@click.option('-l', '--log-level', help='set logging level', default='INFO')
```

Comment:
The help message should maybe list the available logging levels (excluding `NOTSET`), so people can use this without having to look up the levels in the Python documentation.